starcheck/calc_ccd_temps.py

```diff
@@ -294,8 +305,8 @@ def get_week_states(tstart, tstop, bs_cmds, tlm):
     """
     Make states from last available telemetry through the end of the backstop commands
 
-    :param tstart: start time from first backstop command
-    :param tstop: stop time from last backstop command
+    :param tstart: start time from RLTT if available else first backstop command
```

Comment:
This arg name is now quite misleading since it has nothing to do with the "start" of anything.  It is really the "end" of something, namely the end of running loads.  So I'd suggest changing it to `t_rltt`.  I assume this function is not called directly from anywhere outside of this same module.

Reply:
Well, it is the start time of the interval over which we are getting week states.

Reply:
The interval for week states is `(init_tlm_time, schedule_stop_time)`, so `tstart` is somewhere in the middle.  All this stuff is confusing, so it is really important to be clear with variable names and comments.